src/Servers/Kestrel/Core/src/Internal/Http2/IHttp2StreamLifetimeHandler.cs

```diff
@@ -6,5 +6,7 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Core.Internal.Http2
     internal interface IHttp2StreamLifetimeHandler
     {
         void OnStreamCompleted(Http2Stream stream);
+        void IncrementActiveStreamCount();
```

Comment:
I think these should probably have "client" in the name somehow cause there are two counts.

Reply:
Or at least add a doc comment